### ISSUE TITLE
Update checkout instructions to include grunt-cli

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#527]
As reported in https://groups.google.com/forum/#!topic/developing-interactive-simulations-in-html5/XtuHyTxVBxg

On Thursday, April 9, 2015 at 1:59:21 AM UTC-7, Balaji Kamble wrote:
> Above given error comes please let me know the solution for that
I had this same error after following the PhET Development Overview today.

It was resolved using this command:

sudo npm install -g grunt-cli

(Please see this stack-overflow post:
http://stackoverflow.com/questions/10667381/node-package-grunt-installed-but-not-available
In short, "The command line tools are not included with the latest version of Grunt (0.4 at time of writing) instead you need to install them separately.".)

Perhaps a note can be added to the PhET Development Overview telling new developers to install grunt cli tools separately?

Perhaps we should add this to our instructions?

**Comment by samreid** [comment]
It looks as if bit.ly/phet-development-overview the "deprecated development overview" doesn't point to the non-deprecated one.  But is the non-deprecated one group-read-only?

**Comment by samreid** [comment]
It looks like the new one is here: bit.ly/phet-html5-development-overview

**Comment by samreid** [comment]
I pointed to the new version, but cannot edit this file because I do not have write access.  @oliver-phet or @amanda-phet can you please grant @samreid @jbphet @pixelzoom @zepumph @jonathanolson @jessegreenberg write access to this document?

**Comment by oliver-phet** [comment]
@samreid I just allowed edit access to the people you mentioned.

**Comment by samreid** [comment]
Done and noted in https://groups.google.com/forum/#!topic/developing-interactive-simulations-in-html5/XtuHyTxVBxg

Closing.
